.ci/RELEASE_NOTES_GUIDE.md

```diff
@@ -6,162 +6,167 @@ The Magician bot has the ability to copy specifically formatted release notes
-compute: added `foo_bar` field to `google_compute_foo` resource
+compute: added `foo_bar` field to `google_compute_foo`
 ```
 ~~~
-**Bugs:** fixing existing resources
 
+* `release-note:bug`
 ~~~
 ```release-note:bug
-container: fixed perma-diff in `google_container_cluster`
+compute: fixed perma-diff on `region` field in `google_compute_foo` resource
 ```
 ~~~
 
-**Breaking Changes:** changes to existing resources that may require users to change their config
+* `release-note:note`
+~~~
+```release-note:note
+updated Bigtable go client version from 1.13 to 1.16
+```
+~~~
 
+* `release-note:new-resource`
 ~~~
-```release-note:breaking-change
-project: made `iam_policy` authoritative
+```release-note:new-resource
+`google_compute_new_resource`
 ```
 ~~~
 
-**Deprecations:** Announce deprecations when fields/resources are marked as deprecated, not removed
+* `release-note:new-datasource`
+~~~
+```release-note:new-resource
+`google_compute_new_datasource`
+```
+~~~
 
+* `release-note:deprecation`
 ~~~
-``` release-note:deprecation
-container: deprecated `region` and `zone` on `google_container_unicorn`. Use `location` instead.
+```release-note:deprecation
+compute: deprecated `region` field for `google_compute_foo` - use location instead.
 ```
 ~~~
 
-**New Resources And Datasources:**
-(note no service name or *New Resource* tag)
+* `release-note:breaking-change`
+~~~
+```release-note:breaking-change
+compute: changeed `region` field for `google_compute_foo` to be required
+```
+~~~
 
+* `release-note:none`  
 ~~~
-```release-note:new-resource
-`google_compute_new_resource`
+```release-note:none
+(change description can be left empty, or can be optionally filled with an explanation of why the PR is not user-facing)
 ```
 ~~~
 
+### Beta-only changes
+
+To qualify that a change is specific to the beta provider add `(beta)`
+at the end of the release note.
+This will notify whoever is generating the release notes omit the note from changelogs for the ga release.
+
 ~~~
-```release-note:new-datasource
-`google_compute_new_datasource`
+```release-note:enhancement
+compute: added `foo_bar` field to `google_compute_foo` (beta)
 ```
 ~~~
 
-**Notes:** General tag for things that don’t have changes in provider but may be important to users. Syntax is slightly more flexible here. 
+### Revert changes
 
-```release-note:note
-Starting on Nov 1, 2019, Cloud Functions API will be private by default. Add appropriate bindings through `google_cloud_function_iam_*` resources to manage privileges for `google_cloud_function` resources created by Terraform.
+Copy the original PR release note block and add `(revert)` at the end.
+This will let whoever is generating the release notes omit the original PR note from chagnelogs
+
+~~~
+```release-note:enhancement
+compute: added `foo_bar` field to `google_compute_foo` (revert)
+```
+~~~
+
+### Multiple code blocks per PR
+
+You can have multiple code blocks to have multiple release notes per PR, i.e.
```

Comment:
I think this change would mean that the part after the example could be removed.

```suggestion
Multiple release notes in a PR should be represented with one code block per release note, separated by a newline:
```